### DISCUSSION TITLE
Config React app route to run Next.js app

### DISCUSSION THREAD

**Discussion by bhuvaneshdesai** [vercel/next.js#29228] [Help] [answered]
Hi,
I already have a react app, I want to make a particular route in react app as a separate next.js app for SEO.

please can you tell me how to do or where to start from

**Comment by delbaoliveira** [comment, marked as answer]
Hi @bhuvaneshdesai 👋🏼 

You can take the Next.js incremental adoption approach. You may find these two pages helpful: 

- https://nextjs.org/blog/incremental-adoption
- https://nextjs.org/docs/migrating/incremental-adoption
